pumi/pumi_ghost.cc

```diff
@@ -400,7 +400,7 @@ void pumi_ghost_create(pMesh m, Ghosting* plan)
   std::vector<apf::Field*> frozen_fields;
   for (int i=0; i<m->countFields(); ++i)
   {
-    pField f = m->getField(i);
+    apf::Field * f = m->getField(i);
```

Comment:
I should revert this, it crept in while I was working on an issue with freezing fields with FieldDataOf<T> where T != double.

Reply:
I think you should keep the new version since it is consistent with the rest of the (non pumi) codebase